frontend/taipy-gui/src/context/wsUtils.ts

```diff
@@ -3,7 +3,23 @@ import { v4 as uuidv4 } from "uuid";
 
 export const TAIPY_CLIENT_ID = "TaipyClientId";
 
-export type WsMessageType = "A" | "U" | "DU" | "MU" | "RU" | "AL" | "BL" | "NA" | "ID" | "MS" | "DF" | "PR" | "ACK" | "GMC" | "GVS";
+export type WsMessageType =
```

Comment:
Now that this is getting quite long, I wish we have a one-line comment that quicky uncyphers the type string...

Reply:
I wish so too but so far there is no way around this